frontend/src/styles.less

```diff
@@ -10,11 +10,14 @@
 
 // taiga-ui customization
 :root {
-  --top-bar-height: 3.9375rem;
+  --top-bar-height: 5rem;
```

Comment:
potrzebne jest to powiększenie? na mobilnym urządzeniu sprawdzałes czy nie będzie zbyt duże?